src/support_sphere/lib/presentation/pages/main_app/resource/resource_main_body.dart

```diff
@@ -148,6 +149,8 @@ class AllResourcesTab extends StatelessWidget {
             child: Text("No resources found"),
           );
         }
+        // TODO: Figure out how this can be updated as user add and delete resources
```

Comment:
Issue for this todo?

Reply:
Same with this. This is actually now have been implemented.